avaframe/com1DFAPy/Tcpu.py

```diff
@@ -0,0 +1,145 @@
+# import pyximport
+# pyximport.install()
+
+import logging
+import time
+import numpy as np
+import math
+import copy
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+# Local imports
+import avaframe.in2Trans.ascUtils as IOf
+import avaframe.in3Utils.geoTrans as geoTrans
+from avaframe.out3Plot.plotUtils import *
+import avaframe.com1DFAPy.timeDiscretizations as tD
+import avaframe.com1DFAPy.DFAtools as DFAtls
+import avaframe.com1DFAPy.SPHfunctions as SPH
+import avaframe.com1DFAPy.frictionLaws as fricLaws
+
+from SPHfunctionsCython import *
+# import avaframe.in2Trans.shpConversion as shpConv
+# import avaframe.in2Trans.ascUtils as IOf
+# from avaframe.DFAkernel.setParam import *
+
+# create local logger
+log = logging.getLogger(__name__)
+
+slopeAnglex = 40*math.pi/180
+slopeAngley = 20*math.pi/180
```

Comment:
What is this for?

Reply:
I will remove those functions from the git repository